langchain/src/chains/question_answering/stuff_prompts.ts

```diff
@@ -1,8 +1,24 @@
 /* eslint-disable */
-import { PromptTemplate } from "../../prompts/index.js";
+import { ChatPromptTemplate, PromptTemplate } from "../../prompts/index.js";
 
 export const DEFAULT_QA_PROMPT = new PromptTemplate({
   template:
     "Use the following pieces of context to answer the question at the end. If you don't know the answer, just say that you don't know, don't try to make up an answer.\n\n{context}\n\nQuestion: {question}\nHelpful Answer:",
   inputVariables: ["context", "question"],
 });
+
+const chatTemplateSystem = `Use the following pieces of context to answer any user questions. If you don't know the answer, just say that you don't know, don't try to make up an answer
+{context}`;
+const chatPromptSystem = PromptTemplate.fromTemplate(chatTemplateSystem);
+const chatPromptUser = PromptTemplate.fromTemplate("{question}");
+
+export const DEFAULT_CHAT_QA_PROMPT = ChatPromptTemplate.fromPromptMessages([
+  {
+    role: "system",
+    message: chatPromptSystem,
```

Comment:
How about defining them inline here , ie. 
```
{
     role: "system",
     message: PromptTemplate.fromTemplate(chatTemplateSystem)
}
```

Obviously not important for the code here, I'm thinking more about the example we want to set for people creating their own prompts